packages/slopShell/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=slopShell
+pkgver=1
+_SLOP_DIR="/opt/$pkgname-$pkgver/"
+pkgrel=1
+pkgdesc='PHP based backdoor/webshell with C2 capabilities'
+arch=('any')
+groups=('blackarch', 'blackarch-backdoor')
+url='https://github.com/oldkingcone/slopShell'
+license=('Apache License 2.0')
+depends=('postgresql-server', 'php', 'nginx', 'postgresql', 'php-pear', 'php-fpm')
+source=("https://github.com/oldkingcone/slopShell.git")
+# this includes .git directory
+full_sha512sums=('436AE54334F364C8ACC2F2ADF7B85EE2CD8C0DC462A7B17D75197AD382DBC19F')
+# without .git directory
+sha512sums=('4A0F0EAADF1EEE376148847CC4C5B49FCFD494192C2F059F3BA5ECB499E2C41B')
+
+prepare() {
+if ! [ -e $_SLOP_DIR ];then
+    sudo git clone $url $_SLOP_DIR && sudo chown -R $( whoami ):postgres $_SLOP_DIR
```

Comment:
use `cd $pkgname` else you'll clone two times

see template https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git

Reply:
You don't need to clone, the build system will take care for you